blacklight-cornell/app/views/catalog/_call_number_browse_links.html.erb

```diff
@@ -22,7 +22,7 @@
 
 		<div class="browse-buttons">
 	      <% callnumbers.each do |callnumber| %>
-	          <%= link_to callnumber, '/browse?' + {:browse_type => 'Call-Number', :authq => callnumber}.to_query, {:class => "btn btn-cul mb-2"} %>
+			<%= link_to callnumber, browse_index_path(authq: callnumber, browse_type: "Call-Number", start: 0), {:class => "btn btn-cul my-1"} %>
```

Comment:
I see that `start: 0` was added here and in a few other places. Does this change anything?